markup/tree-folders-selectable.html

```diff
@@ -12,7 +12,7 @@
 			</button>
 		</div>
 		<ul class="tree-branch-children" role="group"></ul>
-		<div class="tree-loader" role="alert">Loading...</div>
+		<div class="tree-loader"><span role="alert">Loading...</span></div>
```

Comment:
I was trying to make it consistent and always you use a `span`. You are right the `div`'s don't need to change to pass.